quarkus/runtime/src/test/java/org/keycloak/provider/quarkus/ConfigurationTest.java

```diff
@@ -192,12 +192,23 @@ public void testDatabaseDefaults() {
         System.setProperty("kc.config.args", "--db=h2-file");
         SmallRyeConfig config = createConfig();
         assertEquals(QuarkusH2Dialect.class.getName(), config.getConfigValue("quarkus.hibernate-orm.dialect").getValue());
-        assertEquals("jdbc:h2:file:~/data/keycloakdb;;AUTO_SERVER=TRUE", config.getConfigValue("quarkus.datasource.jdbc.url").getValue());
+        assertEquals("jdbc:h2:file:~\\data\\keycloakdb;;AUTO_SERVER=TRUE", config.getConfigValue("quarkus.datasource.jdbc.url").getValue());
```

Comment:
Could you revert this change, please? It should be enough to get the test passing.